src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/furtherevidence/actionfurtherevidence/ActionFurtherEvidenceMidEventHandlerTest.java

```diff
@@ -45,7 +45,7 @@ public class ActionFurtherEvidenceMidEventHandlerTest {
 
     @Before
     public void setUp() {
-        handler = new ActionFurtherEvidenceMidEventHandler(footerService);
+        handler = new ActionFurtherEvidenceMidEventHandler(footerService, true);
```

Comment:
Shouldn't this be false for the majority of the tests? Then reassign handler with true on the post hearing test(s)

Reply:
changed to true